ui/prometheus-plugin/src/components/PromQLEditor.tsx

```diff
@@ -72,7 +73,10 @@ export function PromQLEditor({ completeConfig, datasource, ...rest }: PromQLEdit
     return new PromQLExtension().activateLinter(false).setComplete(completeConfig).asExtension();
   }, [completeConfig]);
 
-  const queryExpr = useReplaceVariablesInString(rest.value);
+  let queryExpr = useReplaceVariablesInString(rest.value);
+  if (queryExpr) {
+    queryExpr = replacePromBuiltinVariables(queryExpr, 12345, 12345); // TODO placeholder values to replace
```

Comment:
Looks like providing proper values for the step parameters here wont be easy, currently I'd need to rely on the timeseries query context but actually these step values are computed independently / before the queries are getting fired, so maybe we should extract them to independant hooks to be reused here? Need to think about it...